### PR TITLE
Use build.properties as default version/image tag

### PR DESCRIPTION
**Pull request by rrati**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/504#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/504#issuecomment-504161324):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-TAG="latest"
+TAG=$(shell cat build.properties | cut -d'=' -f 2)
```
Comment (msenmurugan) [inline Makefile]: SNAPSHOT will be there in the version. Is it fine?

Comment (rrati) [reply Makefile]: IMO, yes.  Our versions/tags should have -SNAPSHOT or some other string to indicate that the build isn't an official release.  We will need to make sure release builds do not have -SNAPSHOT on them.

Comment (msenmurugan) [reply Makefile]: That will automatically take care by Jenkins CI job

Comment (rrati) [reply Makefile]: That's what I has hoping for. :)

Comment (msenmurugan) [reply Makefile]: ok. you can merge this.. Will change the cloud build accordingly then..

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/504#pullrequestreview-252510161" title="LGTM">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/504#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: #504 failed to apply on top of branch "master":
```Applying: Use build.properties as default version/image tag
error: Failed to merge in the changes.
Using index info to reconstruct a base tree...
M	Makefile
Falling back to patching base and 3-way merge...
Auto-merging Makefile
CONFLICT (content): Merge conflict in Makefile
Patch failed at 0001 Use build.properties as default version/image tag

```

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/504#issuecomment-504161324):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**